src/component/MovieCardListRefetch.js

```diff
@@ -0,0 +1,23 @@
+import MovieCard from "./MovieCard";
+
+/**
+ * cardDatas : [
+ *  {
+ *    href : 영화 상세페이지 url
+ *    src : 영화 썸네일 url
+ *    title : 영화 이름
+ *    alt : 영화 썸네일 alt
+ *    score : 영화 점수
+ *  }
+ * ]
+ */
+export default ({ cardDatas }) => {
+  const ul = document.querySelector("ul");
+
+  cardDatas.map((cardData) => {
+    const { href, src, title, alt, score } = cardData;
+    const card = MovieCard({ href, src, title, alt, score });
+
+    ul.append(card.dom);
+  });
```

Comment:
이 경우, cardDates의 갯수만큼 렌더링이 발생합니다.
append가 계속 실행되니까요!

https://developer.mozilla.org/ko/docs/Web/API/DocumentFragment 를 활용해서 이런식으로 표현해주면 좋겠네요.

```suggestion
  const ul = document.querySelector("ul");
  const fragment = new DocumentFragement()
  cardDatas.map(MovieCard).forEach(({ dom }) => fragment.appendChild(dom));
  ul.appendChild(fragment);
```

Reply:
DocumentFragement 처음알게 됐네요. 수정했습니다.